DQMServices/Core/src/DQMStore.cc

```diff
@@ -2805,60 +2805,6 @@ namespace dqm::dqmstoreimpl {
     return true;
   }
 
-  //////////////////////////////////////////////////////////////////////
-  //////////////////////////////////////////////////////////////////////
-  //////////////////////////////////////////////////////////////////////
-  /// delete directory and all contents;
-  /// delete directory (all contents + subfolders);
-  void DQMStore::rmdir(std::string const& path) {
-    std::string clean;
-    std::string const* cleaned = nullptr;
-    cleanTrailingSlashes(path, clean, cleaned);
-    MonitorElement proto(cleaned, std::string());
-
-    auto e = data_.end();
-    auto i = data_.lower_bound(proto);
-    while (i != e && isSubdirectory(*cleaned, *i->data_.dirname))
-      data_.erase(i++);
-
-    auto de = dirs_.end();
-    auto di = dirs_.lower_bound(*cleaned);
-    while (di != de && isSubdirectory(*cleaned, *di))
-      dirs_.erase(di++);
-  }
-
-  /// remove all monitoring elements from directory;
-  void DQMStore::removeContents(std::string const& dir) {
-    MonitorElement proto(&dir, std::string());
-    auto e = data_.end();
-    auto i = data_.lower_bound(proto);
-    while (i != e && isSubdirectory(dir, *i->data_.dirname))
-      if (dir == *i->data_.dirname)
-        data_.erase(i++);
-      else
-        ++i;
-  }
-
-  /// erase all monitoring elements in current directory (not including subfolders);
-  void DQMStore::removeContents() { removeContents(pwd_); }
-
-  /// erase monitoring element in current directory
-  /// (opposite of book1D,2D,etc. action);
-  void DQMStore::removeElement(std::string const& name) { removeElement(pwd_, name); }
-
-  /// remove monitoring element from directory;
-  /// if warning = true, print message if element does not exist
-  void DQMStore::removeElement(std::string const& dir, std::string const& name, bool const warning /* = true */) {
-    MonitorElement proto(&dir, name);
-    auto pos = data_.find(proto);
-    if (pos != data_.end())
-      data_.erase(pos);
-    else if (warning) {
-      std::cout << "DQMStore: WARNING: attempt to remove non-existent"
```

Comment:
@fabiocos I don't fully understand your comment in a parallel thread, were you testing the old or the new code?

The code that makes the messages you show is deleted here.